Unix/installbuilder/GNUmakefile

```diff
@@ -133,25 +133,26 @@ ifneq ($(PF),Darwin)
 
 	mkdir -p $(PACKAGE_DIR)
 	cp $(OUTPUTDIR)/release/omi-* $(PACKAGE_DIR)
+endif # ifneq ($(PF),Darwin)
 
-else # ifneq ($(PF),Darwin)
-
+ifeq ($(PF),Darwin)
 	# We need a raw copy of omicli (for testing on the Mac platform)
 	mkdir -p $(PACKAGE_DIR)
 	cp $(OUTPUTDIR)/bin/omicli $(PACKAGE_DIR)
 
-endif # ifneq ($(PF),Darwin)
-
-	# We also need a raw copy of libmi shared library. If libmi_origin exists
-	# use that and rename it libmi.
+	# copy the libmi_origin.dylib to libmi.dylib 
+	cp $(OUTPUTDIR)/lib/libmi_origin.dylib $(PACKAGE_DIR)/libmi.dylib
+else #($(PF),Darwin) 
```

Comment:
Prefer a full comment after the 'else'. In this case, `ifeq`. `ifeq ($(PF),Darwin)` is a lot different than `ifneq ($(PF),Darwin)`.